scripts/Muon/GUI/Common/results_tab_widget/results_tab_presenter.py

```diff
@@ -53,9 +56,14 @@ def on_output_results_request(self):
         log_selection = self.view.selected_log_values()
         try:
             self.model.create_results_table(log_selection, results_selection)
+            QMetaObject.invokeMethod(self, "_notify_results_table_created")
```

Comment:
do we want this in the presenter?

Reply:
I think the presenter is the best place for this because it is a communication to one of the other tabs